batchflow/notifier.py

```diff
@@ -387,6 +387,7 @@ def create_description(self, iteration):
 
     def create_message(self, iteration, description):
         """ Combine timestamp, iteration and description into one string message. """
+        iteration = iteration//self.frequency
```

Comment:
Why?

Reply:
A timestamp list saves information by frequency, and when the frequency is more than 1 iteration, we have more iterations than timestamps. To catch the right element in a timestamps list, we need to get the right index.

Reply:
```suggestion
        iteration = iteration // self.frequency
```

Reply:
@cdtn, applied